### ISSUE TITLE
tvOS target

### ISSUE THREAD

**Issue by milobird** [vadymmarkov/Beethoven#59]
I'd love to use this on tvOS. Apart from InputSignalTracker is there anything you'd have to cut? I'm guessing it shouldn't be too tricky to add a tvOS target, you probably just didn't get around to it (yet?) 🙏

**Comment by vadymmarkov** [comment]
Yes, you're right. This library was meant to be used on iOS initially, but could be ported to tvOS I guess, by using conditional compilation statements. I'd be happy to merge a PR if you want to try it out 😊
